package.json

```diff
@@ -59,7 +59,7 @@
     "eslint-plugin-n": "17.11.1",
     "eslint-plugin-no-only-tests": "3.3.0",
     "eslint-plugin-no-secrets": "1.0.2",
-    "eslint-plugin-sonarjs": "1.0.4",
+    "eslint-plugin-sonarjs": "^2.0.4",
```

Comment:
just noticed we have a `^` on this and other peer deps, everything should be fixed to a specific version